### PR TITLE
Add input mask to textfield for phone numbers

### PR DESCRIPTION
**Pull request by cisacke**

### CODE REVIEW

**Comment by cisacke** [comment]
@patrickkim @drewdrewthis adding an input mask for phone numbers, should be a quick 👀  when you have a second. Thanks!

**Comment by patrickkim** [comment]
👀

File: src/molecules/formfields/TextField/index.js [hunk 1]
```diff
@@ -35,13 +37,23 @@ function TextField( props ) {
...
           </label>
         }
-
-        <input
-          className={styles.input}
-          type='text'
-          placeholder={placeholder}
-          {...input}
-        />
+        { phone ?
```
Comment (patrickkim) [inline src/molecules/formfields/TextField/index.js]: Mask seems useful, I think this should be switch out on mask and have the mask pattern and char as an available prop on TextField

**Comment by cisacke** [comment]
@patrickkim made those changes

**Pull request merged**